XenAdmin/Dialogs/LegalNoticesDialog.resx

```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-
-    There are any number of "resheader" rows that contain simple
+                

```

Comment:
I presume all this extra whitespace is Visual Studio messing about?

Reply:
yes this was automatic. I didn't even touch the code. I simply used VS' form for changing strings. Tried this a few times with the same result.